packages/experiments/src/demo/GettingStartedPage.tsx

```diff
@@ -1,103 +1,17 @@
-          node_modules/office-ui-fabric-react.
-        </p>
-
-        <p>
-          The library includes commonjs entry points under the lib folder. To use a control, you should be able to import it and use it in
-          your render method. Note that wrapping your application in the Fabric component is required to support RTL, keyboard focus and
-          other features.
-        </p>
-
-        <div className="ms-GettingStartedPage-code">
-          <Highlight className="typescript">{`
-import * as React from 'react';
-import * as ReactDOM from 'react-dom';
-import { Fabric } from 'office-ui-fabric-react/lib/Fabric';
-import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
-
-const MyPage = () => (<Fabric><DefaultButton>I am a button.</DefaultButton></Fabric>);
-
-ReactDOM.render(<MyPage />, document.body.firstChild);`}</Highlight>
-        </div>
-
-        <h2>Notes on module vs path-based imports</h2>
-        <p>
-          While it is possible to import all components as named imports from the main module entry point, it is not recommended to do so
-          without using a bundler that supports es6 tree shaking. In other words, if you import the Button component like this: }
-        </p>
-
-        <div className="ms-GettingStartedPage-code">
-          <Highlight className="typescript">{`import { Button } from 'office-ui-fabric-react';`}</Highlight>
-        </div>
-
-        <p>
-          ...this would work, but then unless you are using a tree-shaking bundler such as Rollup.js or Webpack 2, Webpack will assume you
-          want every module exported from the main entry file to be included in your final bundle, which produces unnecessary large bundles
-          and slows your page load down. Instead you can import the specific paths to trim down your bundle size:
-        </p>
-
-        <div className="ms-GettingStartedPage-code">
-          <Highlight className="typescript">{`
-import { Button } from 'office-ui-fabric-react/lib/Button';
-import { Dropdown } from 'office-ui-fabric-react/lib/Dropdown';
-import { List } from 'office-ui-fabric-react/lib/List';`}</Highlight>
-        </div>
-
-        <h2>Using an AMD bundler like r.js</h2>
-
-        <p>
-          If your project relies on AMD modules, they are dropped in the lib-amd folder. You will need to set up your bundler to handle the
-          imports correctly. This may require you to symlink or copy the folder into your pre-bundle location.
-        </p>
-      </div>
-    );
-  }
-}
+import { PageMarkdown } from '@uifabric/example-app-base';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+const rootClass = mergeStyles({
+  marginTop: -20,
+  padding: 40
+});
+
+export const GettingStartedPage: React.StatelessComponent = () => {
+  return (
+    <div className={rootClass}>
+      {/* tslint:disable-next-line:deprecation */}
+      <PageMarkdown>{require<string>('!raw-loader!../../README.md')}</PageMarkdown>;
```

Comment:
A few of the Getting Started pages have an errant `;` in the content. Do a quick sweep for these and you should be good.
```suggestion
      <PageMarkdown>{require<string>('!raw-loader!../../README.md')}</PageMarkdown>
```

Reply:
Good catch, thanks!